### PR TITLE
Remove unnecessary changes

### PR DESCRIPTION
**Pull request by LevchenkoS**
Unnecessary changes, transferred to SDL with RC merge, have been removed.

Related to: [APPLINK-31599](https://adc.luxoft.com/jira/browse/APPLINK-31599)
##
@LuxoftAKutsan, @okozlovlux, please review.

### CODE REVIEW

File: src/components/application_manager/src/application_impl.cc [hunk 1]
```diff
@@ -1047,6 +969,76 @@ void ApplicationImpl::UnsubscribeFromSoftButtons(int32_t cmd_id) {
...
+    list_files_in_none_count_ = 0;
+  }
+  LOG4CXX_INFO(logger_, "hmi_level = " << hmi_level);
+  hmi_level_ = hmi_level;
+  usage_report_.RecordHmiStateChanged(hmi_level);
+}
+
+bool ApplicationImpl::SubscribeToInteriorVehicleData(
+    smart_objects::SmartObject module) {
+  // size_t old_size = subscribed_interior_vehicle_data_.size();
```
Comment (okozlovlux) [inline src/components/application_manager/src/application_impl.cc]: @LevchenkoS , need this comment?
Check other files

File: src/3rd_party-static/MessageBroker/src/example/MessageBrokerControllerAVA.hpp [hunk 2]
```diff
@@ -65,4 +65,4 @@ namespace NsMessageBroker
 
    };
 }/* namespace NsMessageBroker */
-#endif /* MB_CONTROLLERAVA_H */
+#endif /* MB_CONTROLLERAVA_H */
```
Comment (LuxoftAKutsan) [inline src/3rd_party-static/MessageBroker/src/example/MessageBrokerControllerAVA.hpp]: @LevchenkoS please ensure that ending of files are correct.

File: src/3rd_party-static/jsoncpp/src/lib_json/json_reader.cpp [hunk 3]
```diff
@@ -576,7 +576,7 @@ Reader::decodeNumber( Token &token )
                                                    : Value::maxLargestUInt;
    Value::LargestUInt threshold = maxIntegerValue / 10;
    Value::UInt lastDigitThreshold = Value::UInt( maxIntegerValue % 10 );
-   assert( lastDigitThreshold >=0  &&  lastDigitThreshold <= 9 );
```
Comment (LuxoftAKutsan) [inline src/3rd_party-static/jsoncpp/src/lib_json/json_reader.cpp]: @LevchenkoS are you sure that you need to delete it?

File: src/3rd_party/apache-log4cxx-0.10.0/autom4te.cache/requests [hunk 4]
```diff
@@ -15,63 +15,63 @@
                         'configure.in'
                       ],
                       {
+                        'AM_MAKEFILE_INCLUDE' => 1,
```
Comment (LuxoftAKutsan) [inline src/3rd_party/apache-log4cxx-0.10.0/autom4te.cache/requests]: @LevchenkoS I suppose that this file should not be changed.

File: src/appMain/CMakeLists.txt [hunk 5]
```diff
@@ -209,47 +209,31 @@ if (${QT_HMI})
...
-  #file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/hmi_link
-  #  "${CMAKE_BINARY_DIR}/src/components/HMI/index.html")
 endif ()
 
 # Install rules
 install(TARGETS ${PROJECT} DESTINATION bin)
+
 install(
   FILES log4cxx.properties audio.8bit.wav test.txt smartDeviceLink.ini
-    hmi_capabilities.json sdl_preloaded_pt.json
```
Comment (LuxoftAKutsan) [inline src/appMain/CMakeLists.txt]: @LevchenkoS preloaded should not be copied to bin?

File: src/components/application_manager/CMakeLists.txt [hunk 6]
```diff
@@ -114,24 +114,6 @@ file (GLOB MOBILE_COMMANDS_SOURCES
 )
 
 if (NOT REMOTE_CONTROL)    
-    list(REMOVE_ITEM MOBILE_COMMANDS_SOURCES
-        ${COMMANDS_SOURCE_DIR}/mobile/set_interior_vehicle_data_request.cc)
-    list(REMOVE_ITEM MOBILE_COMMANDS_SOURCES
-        ${COMMANDS_SOURCE_DIR}/mobile/get_interior_vehicle_data_response.cc)
-    list(REMOVE_ITEM MOBILE_COMMANDS_SOURCES
-        ${COMMANDS_SOURCE_DIR}/mobile/get_interior_vehicle_data_capabilities_response.cc)
```
Comment (LuxoftAKutsan) [inline src/components/application_manager/CMakeLists.txt]: @LevchenkoS isn't this changes related to https://github.com/smartdevicelink/sdl_core/pull/1192 ?

Comment (LuxoftAKutsan) [inline src/components/application_manager/CMakeLists.txt]: @LevchenkoS please make sure that this pull requests will be interdependent to have possibility to restore each one of them independently

Comment (LevchenkoS) [inline src/components/application_manager/CMakeLists.txt]: @LuxoftAKutsan, this pull request is not related to #1192

File: src/3rd_party/apache-log4cxx-0.10.0/autom4te.cache/requests (hunk 4 above)
Comment (LevchenkoS) [inline src/3rd_party/apache-log4cxx-0.10.0/autom4te.cache/requests]: @LuxoftAKutsan, Yes, but it has been changed during RC merge. So this PR will revert this changes.

File: src/appMain/CMakeLists.txt (hunk 5 above)
Comment (LevchenkoS) [inline src/appMain/CMakeLists.txt]: @LuxoftAKutsan, yes, it should be copied, and it would be. It's done in the next `if` statement.

File: src/3rd_party-static/jsoncpp/src/lib_json/json_reader.cpp (hunk 3 above)
Comment (LevchenkoS) [inline src/3rd_party-static/jsoncpp/src/lib_json/json_reader.cpp]: @LuxoftAKutsan, yes, because `lastDigitThreshold` just can't be lower then zero, because it's type is `Value::UInt`
